doc/technotes/errorHandling.rst

```diff
@@ -0,0 +1,190 @@
+.. _readme-errorHandling:
+
+==============
+Error Handling
+==============
+
+This README describes the error handling feature as currently implemented.
+
+
+Errors
+------
+A base class ``Error`` is defined in the Chapel standard modules.
+
+::
+
+  class Error {
+    var msg: string;
+
+    proc Error(_msg: string) {
+      msg = _msg;
+    }
+  }
+
+Users may use the provided ``Error`` directly, or create their own error
+hierarchy with ``Error`` at its root.
+
+::
+
+  class MyError: Error {}
+
+  class MyIntError: MyError {
+    var i: int;
+  }
+
+Chapel does not provide a standard set of ``Error`` classes.
+
+
+Throwing Errors
+---------------
+``throw`` statements throw errors from a function to its callee.
+
+::
+
+  throw new Error("with a message");
```

Comment:
I think the two sections in this code block need to have some exposition splitting them up. That could be in comments or it could be 2 code blocks. It should say something like

You can throw a newly created error:


Or, you can throw an error that is already stored in a variable: